projects/fusion-ui/components/table/v3/components/table-basic/table-basic.component.html

```diff
@@ -77,4 +85,21 @@
         </tr>
     </ng-template>
     <!--Recursive template for child rows render END-->
+    <!--additional row-span rows-->
+    <ng-template #multiRowColumns let-row let-rowIndex="rowIndex" let-rowSpanData=rowSpanData>
```

Comment:
and this is a template for rendering "spanned" rows. getRowspanIndexes return an array like [1,2...maxspan].
(without "0" zero) because it used in the main row render